csunplugged/utils/resource_valid_test_configurations.py

```diff
@@ -4,7 +4,7 @@
 from utils.bool_to_yes_no import bool_to_yes_no
 
 
-def resource_valid_test_configurations(valid_options):
+def resource_valid_test_configurations(valid_options, header_text=True):
```

Comment:
new parameter needs to be added to docstring args

Reply:
Fixed.